systray_darwin.go

```diff
@@ -21,7 +21,7 @@ import (
 // .ico/.jpg/.png for other platforms.
 func SetTemplateIcon(templateIconBytes []byte, regularIconBytes []byte) {
```

Comment:
should we think about try to PR a change into the fyne repo? possibly add a func for SetTemplateIconWithColor or something like that? For windows / linux it could just pass the parameters to SetTemplateIcon.